q2_sample_classifier/assets/index.html

```diff
@@ -70,7 +70,22 @@
       <a href="feature_importance.tsv">
         <p>Download as tsv</p>
       </a>
-      {{ importances }}
+    </div>
+    <div class="panel-group" id="accordion" role="tablist" aria-multiselectable="true">
```

Comment:
Can you nest this panel `div` into the `<div class="col-lg-12">` above? That way the panel will fit inside of the grid container.

before:
<img width="1680" alt="screen shot 2017-09-06 at 11 37 03 am" src="https://user-images.githubusercontent.com/274668/30128497-bcfd2bb0-92f7-11e7-905f-569f4cef3742.png">

after:
<img width="1680" alt="screen shot 2017-09-06 at 11 37 27 am" src="https://user-images.githubusercontent.com/274668/30128510-ca598e98-92f7-11e7-9b3c-4a3f186cc548.png">